functests/execpublicassets.py

```diff
@@ -102,9 +114,12 @@ def test_public_asset_create_simple_hash(self):
             True,
             msg="Asset is not public",
         )
-        count = self.arch.publicevents.count(asset_id=asset["identity"])
+        identity = asset["identity"]
+        public = self.arch.Public
+        asset_id = f"{self.url}/{ROOT}/{ASSETS_SUBPATH}/public{identity}"
```

Comment:
yes - this is temporary because the public endpoints and publicurl stuff was not implemented.

The next PR fixes this - this PR is big enough without adding the code for the publicurl handling as well but it is writtedn and tested 
If you want I can incoporate ist now...

Reply:
next PR is merged so the issue with SUBPATH in public url should be fixed